_topic_map.yml

```diff
@@ -620,18 +620,29 @@ Topics:
         File: ruby_on_rails
       - Name: Setting Up a Nexus Mirror
         File: maven_tutorial
-  - Name: Opening a Remote Shell to Containers
-    File: ssh_environment
-  - Name: Templates
-    File: templates
-  - Name: Service Accounts
-    File: service_accounts
   - Name: Builds
-    File: builds
-  - Name: Managing Images
-    File: managing_images
-  - Name: Quotas and Limit Ranges
-    File: compute_resources
+    Dir: builds
+    Topics:
+      - Name: How Builds Work
+        File: index
+      - Name: Basic Build Operations
+        File: basic_build_operations
+      - Name: Build Inputs
+        File: build_inputs
+      - Name: Build Output
+        File: build_output
```

Comment:
Hmm. I think these are pretty clear and succinct topic titles and I think it's easy to over-complicate things. An input is something that goes into a build and an output is what comes out of that. We could define input/output at the start of their respective topic, but maybe that would be overkill. :woman_shrugging: